stacks-in-depth/nodes-and-miners/mine-testnet-stacks-tokens.md

```diff
@@ -50,35 +50,33 @@ rpcport=18332
 
 #### Start Bitcoin
 
-Finally, start bitcoind as follows:
+Finally, start `bitcoind` as follows (adjust the `conf` path to where it was created in the previous step, i.e. `$HOME/bitcoin.conf`):
 
 ```bash
 bitcoind -conf=$HOME/bitcoin.conf
 ```
 
-:::note It will take a few hours for the node to synchronize with Bitcoin testnet. :::
+**Note:** It will take a few hours for the node to synchronize with Bitcoin Testnet.
 
 While it's syncing, you can track the progress with `bitcoin-cli` or the logfile (will be located where the chainstate is stored, i.e. `/bitcoin/testnet3/debug.log`):
 
 ```bash
 $ bitcoin-cli \
- -rpcconnect=localhost \
+ -rpcconnect=127.0.0.1 \
  -rpcport=18332 \
  -rpcuser=btcuser \
  -rpcpassword=btcpass \
-getblockchaininfo | jq .blocks
-2417570
+  getblockchaininfo | jq .blocks
+2583513
 ```
 
 ***
 
 ### Running a Stacks Blockchain miner
 
-First, download a [stacks blockchain binary](https://github.com/stacks-network/stacks-blockchain/releases/latest), or [build from source](https://github.com/stacksfoundation/miner-docs/blob/main/stacks-blockchain.md#build-and-install-stacks-blockchain-from-source)
+First, download a [stacks blockchain binary](https://github.com/stacks-network/stacks-blockchain/releases/latest), or [build from source](https://github.com/stacksfoundation/miner-docs/blob/testnet/stacks-blockchain.md#build-and-install-stacks-blockchain-from-source) (_there may be some extra requirements to building,_ [_defined here_](https://github.com/stacksfoundation/miner-docs/blob/testnet/prerequisites.md#install-required-packages)).
 
-_There may be some extra requirements to building,_ [_defined here_](https://github.com/stacksfoundation/miner-docs/blob/main/prerequisites.md#install-required-packages)
-
-:::tip It is recommened to use a persistent location for the chainstate, in the steps below we're using `/stacks-blockchain` :::
```

Comment:
these should remain. 
- there are quite a few dependencies required that the link will be helpful in resolving
- the `tip` line is formatted for githbook, so the markdown should be the same as was removed here.

Reply:
They remained, I've just changed their placement (they now are with parentheses after `build from source` - also added them to the bitcoin build step).
![image](https://github.com/stacks-network/docs/assets/157840260/b0dedb36-3ac0-4816-9958-232ce23ce594)
![image](https://github.com/stacks-network/docs/assets/157840260/64c8ce0f-8e27-4653-bfff-47baac736094)